docs/data-sources/connectors.md

```diff
@@ -19,17 +19,20 @@ data "twingate_connectors" "all" {}
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
+### Optional
+
+- `connectors` (Attributes List) List of Connectors (see [below for nested schema](#nestedatt--connectors))
+
 ### Read-Only
 
-- `connectors` (Block List) List of Connectors (see [below for nested schema](#nestedblock--connectors))
-- `id` (String) The ID of this resource.
+- `id` (String) The ID of the data source result
 
-<a id="nestedblock--connectors"></a>
+<a id="nestedatt--connectors"></a>
 ### Nested Schema for `connectors`
 
 Read-Only:
 
-- `id` (String) The ID of the Connector.
+- `id` (String) The ID of the Connector resource.
```

Comment:
why did this change?

Reply:
@alexmensch asked to change like this https://github.com/Twingate/terraform-provider-twingate/pull/328#discussion_r1235479014